guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -19,6 +21,27 @@ ifdef::foreman-deb[]
 endif::[]
 
 ifdef::foreman-el,katello[]
+[id="repositories-centos-stream-9"]
+== CentOS Stream 9
+
+:distribution: el
+:distribution-major-version: 9
+:package-manager: dnf
+
+include::proc_configuring-repositories-el.adoc[]
+
+ifdef::katello[]
++
+. Enable `powertools` repository:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} config-manager --set-enabled powertools
+----
+endif::[]
```

Comment:
We don't need powertools anymore.

@ehelms correct me, but this can be removed from el8 too, as this is katello-agent related and we dropped that?

Reply:
I honestly do not recall what we needed from powertools. Forklift would indicate you are correct here:

```
- name: enable powertools for libdb_cxx used by qpid-cpp-server-linearstore
```

Reply:
Removed :+1: